block_engine/src/block_engine.rs

```diff
@@ -475,11 +436,17 @@ impl BlockEngineRelayerHandler {
 
                     block_engine_stats.increment_flush_elapsed_us(flush_start.elapsed().as_micros() as u64);
                     block_engine_stats.increment_accounts_of_interest_len(accounts_of_interest.cache_size() as u64);
+                    block_engine_stats.increment_programs_of_interest_len(programs_of_interest.cache_size() as u64);
 
-                    block_engine_stats.report(&cluster, &region);
+                    block_engine_stats.report();
                     block_engine_stats = BlockEngineStats::default();
                 }
             }
+
+            block_engine_stats.update_block_engine_packet_sender_len(
+                (Self::BLOCK_ENGINE_PACKET_QUEUE_CAPACITY - block_engine_packet_sender.capacity())
```

Comment:
I think emitting capacity would be useful and easy to tell if the queue is filling up without looking at the code